classes/class-wc-connect-shipping-label.php

```diff
@@ -25,22 +25,27 @@ class WC_Connect_Shipping_Label {
 		protected $payment_methods_store;
 
 		/**
-		 * @var array array of currently unsupported US states
+		 * @var array array of currently supported countries
 		 */
-		private $unsupported_states = array( 'AA', 'AE', 'AP' );
+		private $supported_countries = array( 'US', 'PR' );
```

Comment:
You need to add `'VI'` (US Virgin Islands) too, since it doesn't require a customs form either..